ERMize/topics/Entity/Naming-Identifiers.topic

```diff
@@ -5,14 +5,29 @@
        xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
        title="Naming Identifiers" id="Naming-Identifiers">
 
-    <available-only-for instance="ermize">Standard</available-only-for>
-
+    <available-only-for>Standard</available-only-for>
+    <p>Identifiers are commonly used in conceptual and logical data models. When identifiers are named properly it can provide more clarity about the identifier.
+        Identifiers can also help stakeholders to understand what each attribute or entity represents.
+        By using identifiers, the user can enhance the documentation of the data model.
+    </p>
     <chapter title="Code" id="code">
+        <p>
+            We have added the functionality to add identifiers to entities.
+            See the code snippet below how this is done in code.
+        </p>
+        <note>
+            <p>The user has the ability to add identifiers when no attributes exist in the entity.
+            See code snippet below for an example how this is done.
+            </p>
+            <code-block src="naming-identifiers-without-attributes.txt" />
+        </note>
         <code-block src="naming-identifiers.txt" />
+
     </chapter>
 
     <chapter title="Output" id="output">
-        <note>This is a concept image and not the actual result of the application.</note>
+        <p>This is the output image that's being generated by ERMize based on the code snippet above.</p>
         <img src="naming-identifiers.svg" alt="Entity with a named identifier" />
```

Comment:
Deze afbeelding moet nog aangepast worden naar een afbeelding die gegenereerd is door de API. Dit kan gedaan worden als de functionaliteit toegevoegd is.